bblfsh-dashboard/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
 name: bblfsh-dashboard
-version: 0.1.0
+version: 0.1.1
```

Comment:
What is the purpose of this "number"?
Internal control? Is it used in any other place? Must be updated in every single PR? When must it be updated?

Reply:
Charts are versioned too. Every change in the chart should imply a bump of this version. 

```
$ helm list
NAME            	REVISION	UPDATED                 	STATUS  	CHART                 	NAMESPACE
bblfsh-dashboard	3       	Tue Sep 19 23:40:17 2017	DEPLOYED	bblfsh-dashboard-0.1.0	default  
``` 
Then you can see which version of the chart is deployed